### DISCUSSION TITLE
New Multiple Client Instances Parameter

### DISCUSSION THREAD

**Discussion by GitGoorilla** [chuot/rdio-scanner#270] [General]
Could you explain how this feature works? I'm assuming it's ?id=TG1,TG2,TG3 but I don't get a result from that parameter.

**Comment by chuot** [comment]
Not exactly.

The ?id=instance1 and ?id=instance2 allow to have multiple instances running at the same time, where each of them will remember the talkgroups selection separately across sessions. This is useful if you want to have different bookmarks in your browser so that each instance brings back its own selection.

Before that, only the last selection changes were remembered and you had to reselect the desired talkgroups each time you reopened the web application.
